code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -288,7 +286,7 @@
 	M.adjustOrganLoss(ORGAN_SLOT_LIVER, 0.1)
 	M.adjustToxLoss(-1*REM, 0)
 	for(var/datum/reagent/toxin/R in M.reagents.reagent_list)
```

Comment:
this seems to conflict with what you've said in the OP of the PR, but tbh, I prefer musiver this way (if you do want it to purge all chems, though, make it purge all chems except for syriniver and musiver (itself), or else this will be really, really bad)